packages/website/ts/components/ui/drop_down.tsx

```diff
@@ -8,12 +8,13 @@ const DEFAULT_STYLE = {
 };
 
 interface DropDownProps {
-    hoverActiveNode: React.ReactNode;
+    activeNode: React.ReactNode;
     popoverContent: React.ReactNode;
     anchorOrigin: MaterialUIPosition;
     targetOrigin: MaterialUIPosition;
     style?: React.CSSProperties;
     zDepth?: number;
+    shouldWaitForClickToActivate?: boolean;
```

Comment:
What if the API was `activateEvent: DropdownMouseEvent`, `closeEvent: DropdownMouseEvent` where `type DropdownMouseEvent = 'hover' | 'click'. I guess this isn't strictly necessary, but would be nice.

I feel like the only way to solve the issue on mobile is `closeEvent={utils.isMobile() ? 'click' : 'hover'}`

Implementing the 'onClickOutside' can be a bit difficult. What I've done in the past is make a HOC (https://github.com/fragosti/codenail-app/blob/master/src/HOCs/withOnClickOutside.js), although those aren't cool anymore and people seem to be converging to "renderProps" https://cdb.reacttraining.com/use-a-render-prop-50de598f11ce.

This also seems simple enough https://medium.com/@pitipatdop/little-neat-trick-to-capture-click-outside-react-component-5604830beb7f